SimCalorimetry/HcalTrigPrimProducers/python/hcaltpdigi_cff.py

```diff
@@ -54,3 +54,6 @@
 from Configuration.Eras.Modifier_run3_oxygen_cff import run3_oxygen
 from Configuration.Eras.Era_Run3_2025_UPC_cff import Run3_2025_UPC
 (pp_on_PbPb_run3_2025 | run3_oxygen | Run3_2025_UPC).toModify(HcalTPGCoderULUT, FG_HF_thresholds = [14, 16])
```

Comment:
Not part of this PR, but was this line possibly intended to be
```suggestion
(pp_on_PbPb_run3_2025 | run3_oxygen | run3_upc).toModify(HcalTPGCoderULUT, FG_HF_thresholds = [14, 16])
```
?

The `cms.ModifierChain` (that `Run3_2025_UPC` is) were not intended to be used like this (but I see now this loophole is technically allowed by the boolean expressions of `Modifier`s).

I can open a separate issue about this if that would be better.

Reply:
Hi, Perhaps @stahlleiton can comment on this? Happy to fix if desired.

Reply:
It was made intentionally to separate this particular setting across each year using their era without creating a new modifier exclusively for this. But if this poses an issue, we can change to different year based modifiers (e.g. run3_upc_2025)

Reply:
It would be best to use a `Modifier` instead of `ModifierChain` (maybe best done in a separate PR?)

Reply:
Okay cool, perhaps we can then proceed with merging this PR and then handle that separately?